File: Practica-POO/Semana01/Ejercicio01/Rectangle.cpp
#include <iostream>
using namespace std;

class Rectangle{
    public:
    double base;
    double height;

    Rectangle(double b,double h){
        base = b;
        height = h;
    }

    double CalculateArea(){
        return height*base;
    }

    double CalculatePerimeter(){
        return 2*(height+base);
    }
};

int main(){
    Rectangle rectangle1(1,4);

    double area1 = rectangle1.CalculateArea();
    double perimeter1 = rectangle1.CalculatePerimeter();

    cout<< "El area es: " << area1 << endl;
    cout<< "El perimetro es: " << perimeter1 << endl;
    return 0;


}

File: Practica-POO/Semana01/Ejercicio01/Rectangle.py
class Rectangle:
    def __init__(self,base,height):
        self.base = base
        self.height = height

    def CalculateArea(self):
        return self.base * self.height

    def CalculatePerimeter(self):
        return 2*(self.base+self.height)

rectangle1 = Rectangle(2,3)
area1 = rectangle1.CalculateArea()
perimeter1 = rectangle1.CalculatePerimeter()
print (f"El area es: {area1}")
print(f"El perimtro es: {perimeter1}")

